components/avatar/index.js

```diff
@@ -0,0 +1,14 @@
+export default ({
```

Comment:
Anonymous components should be avoided as they show up as `<undefined ... />` in debuggers and tests. You can use implicitly named arrow functions however:

```jsx
const Avatar = () => /* ... */
export default Avatar
```

Or you could use an explcitly named function:

```jsx
export default function Avatar () {
  /* ... */
}
```